apps/omg_child_chain/lib/omg_child_chain/release_tasks/set_fee_feed_adapter_opts.ex

```diff
@@ -66,7 +65,9 @@ defmodule OMG.ChildChain.ReleaseTasks.SetFeeFeedAdapterOpts do
   end
 
   defp configure_adapter(_not_feed_adapter) do
-    configure_adapter({OMG.ChildChain.Fees.FeedAdapter, opts: []})
+    adapter = {OMG.ChildChain.Fees.FeedAdapter, opts: []}
```

Comment:
this is all getting deleted in the later PR no need to spend time reviewing anything omg_child_chain*/